backend/src/test/java/com/google/sps/UserDatastoreTest.java

```diff
@@ -91,7 +104,7 @@ public void addGetTwoUsers() {
     assertThat(userDatastore.getUserFromUsername(PERSON_B).getUsername()).isEqualTo(PERSON_B);
   }
 
-  @Test
+  @Test(expected = NullPointerException.class)
```

Comment:
I would expect this test method not to throw. Where is the NPE coming from?

Reply:
Since I got rid of the null check in `getUserFromEntity()` and made the parameter @Nonnull, since I'm deliberately making that argument null in getUserFromUsername(), a NullPointerException is thrown. How do I go about changing this?

Reply:
use a non-null username that won't be in the datastore

Reply:
I did add a non-null String username (PERSON_A) that's not in the datastore. getEntity(PERSON_A) returns null, but getUserFromEntity() only accepts non-null entities.